### PR TITLE
Pyic 6736 strategic app journey map

### PR DESCRIPTION
**Pull request by DanCorderIPV**
### What changed

Reinstate appTriage events to journey map

### Why did it change

Strategic app work re-commencing.

This is commit 2 of 4. https://github.com/govuk-one-login/ipv-core-front/pull/1584 must be deployed first.

### Issue tracking

- [PYIC-6736](https://govukverify.atlassian.net/browse/PYIC-6736)

[PYIC-6736]: https://govukverify.atlassian.net/browse/PYIC-6736?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reverification.yaml [hunk 1]
```diff
@@ -82,9 +82,47 @@ states:
...
+          dcmaw:
+            targetJourney: TECHNICAL_ERROR
+            targetState: ERROR
+      appTriageAndroid:
+        targetState: CRI_DCMAW
+        checkFeatureFlag:
+          strategicAppEnabled:
+            targetState: STRATEGIC_APP_TRIAGE
+        checkIfDisabled:
+          dcmawIphone:
```
Comment (blakeyp) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reverification.yaml]: should be just `dcmaw` ?

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml [hunk 2]
```diff
@@ -430,6 +562,19 @@ states:
...
+      next:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+      end:
+        targetJourney: FAILED
+        targetState: FAILED_UPDATE_DETAILS
+      multipleDocCheckPage:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+
```
Comment (blakeyp) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml:463]: Hmm so there's a bunch of different end states for these CoI scenarios which we'll need to handle on the handback from the app (when we get to it)

Comment (DanCorderIPV) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml:463]: Yes, currently I think everywhere that goes down the strategic app route does this.

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reverification.yaml (hunk 1 above)
Comment (DanCorderIPV) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reverification.yaml]: It should

**Review by blakeyp — DISMISSED** [review]

**Review by blakeyp — APPROVED** [review]

**Pull request merged**